textile/features.textile

```diff
@@ -717,8 +717,8 @@ h3(#realtime-presence). RealtimePresence
 ** @(RTP17h)@ Unlike the main @PresenceMap@, which is keyed by "memberKey":#TP3h , the @RTP17@ @PresenceMap@ must be keyed only by @clientId@. (Otherwise, entries associated with old @connectionId@s would never be removed, even if the user deliberately leaves presence).
 ** @(RTP17f)@ The RealtimePresence object should perform automatic re-entry whenever a channel moves into the @ATTACHED@ state. (It does not need to do so for @RTL12@ @ATTACHED@ events received on already-@ATTACHED@ channels).
 ** @(RTP17g)@ Automatic re-entry consists of, for each member of the @RTP17@ internal @PresenceMap@, publishing a @PresenceMessage@ with an @ENTER@ action using the @clientId@, @data@, and @id@ attributes from that member.
-** @(RTP17c)@ This clause and its subclauses have been replaced by @RTP17f@ as of the 2.0 spec.
```

Comment:
https://github.com/ably/specification/pull/120/commits/39f8d2e41234a716d3b6018061342c2941feafe8

Reply:
Thanks! Should we now apply this new guidance to RTP17c (that is, restore its subclauses and mark them as removed)?

Reply:
Oh, I had missed that in relation to `RTP17c`. I would like to defer that to a later PR as the alteration was made outside of the scope of this PR's change (i.e. it was done already in [the target](https://github.com/ably/specification/pull/108)). I've created https://github.com/ably/specification/issues/122 so that this does not get forgotten.